### PR TITLE
Stand chart

### PR DESCRIPTION
**Pull request by pflopez**
- Adds the stand data chart (bottom one on direct impacts)
- Added another 'state' for direct treatment impacts, so we can decouple from the main `treatment` state.
- Moves some code around

### CODE REVIEW

File: src/interface/src/app/treatments/direct-impacts-map/direct-impacts-map.component.ts [hunk 1]
```diff
@@ -38,23 +41,10 @@ export class DirectImpactsMapComponent {
...
-    five: 'delta_5',
-    ten: 'delta_10',
-    fifteen: 'delta_15',
-    twenty: 'delta_20',
-  };
+  readonly labels = YEAR_INTERVAL_LABELS;
 
-  @Input() year: Year = 'cero';
+  readonly variables = YEAR_INTERVAL_PROPERTY;
+  @Input() year: YearInterval = 'cero';
```
Comment (lastminutediorama) [inline src/interface/src/app/treatments/direct-impacts-map/direct-impacts-map.component.ts:47]: I think this should be zero :)

File: src/interface/src/app/treatments/direct-impacts.state.service.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+import { DEFAULT_SLOT, MapMetric, METRICS } from './metrics';
+import { MapGeoJSONFeature } from 'maplibre-gl';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class DirectImpactsStateService {
```
Comment (lastminutediorama) [inline src/interface/src/app/treatments/direct-impacts.state.service.ts:9]: nice, thanks for adding this service

File: src/interface/src/app/treatments/direct-impacts-map/direct-impacts-map.component.ts (hunk 1 above)
Comment (pflopez) [reply src/interface/src/app/treatments/direct-impacts-map/direct-impacts-map.component.ts:47]: 🤦‍♂️  this makes billy Corgan sad, he wore that shirt for nothing.
Thanks!

File: src/interface/src/app/treatments/metrics.ts [hunk 3]
```diff
@@ -86,3 +86,21 @@ export const SLOT_PALETTES: Record<
   orange: ['#FFE0B2', '#FFB74D', '#FF9800', '#F57C00', '#E65100'],
   green: ['#C8E6C9', '#81C784', '#43A047', '#2E7D32', '#1B5E20'],
 };
+
+export type YearInterval = 'cero' | 'five' | 'ten' | 'fifteen' | 'twenty';
+
+export const YEAR_INTERVAL_LABELS: Record<YearInterval, string> = {
+  cero: 'Baseline',
```
Comment (lastminutediorama) [inline src/interface/src/app/treatments/metrics.ts:93]: also "zero" here and in YearInterval above and YEAR_INTERVAL_PROPERTY below

File: src/interface/src/app/treatments/treatments.state.ts [hunk 4]
```diff
@@ -202,7 +196,7 @@ export class TreatmentsState {
             // set active project area if provided
             this.selectProjectArea(projectAreaId);
           } else if (!previousSummary) {
-            // if its the first time loading summary, center the map
+            // if it's the first time loading summary, center the map
```
Comment (lastminutediorama) [inline src/interface/src/app/treatments/treatments.state.ts:199]: :laughing: Billy Corgan would never have caught this.

File: src/interface/src/app/treatments/stand-data-chart/stand-data-chart.component.ts [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+          {
+            data: data,
+            barThickness: 30,
+          },
+        ],
+      } as ChartConfiguration<'bar'>['data'];
+    })
+  );
+
+  private readonly staticBarChartOptions: ChartConfiguration<'bar'>['options'] =
```
Comment (lastminutediorama) [inline src/interface/src/app/treatments/stand-data-chart/stand-data-chart.component.ts:52]: I guess I'll need to catch up on how we do chart.js configs, but sometime in the future, maybe we can coordinate on a way to organize these so we're blending less styles into ts code.

**Review by lastminutediorama — APPROVED** [review]
lgtm -- thanks for starting this one, it also seems like a good basic structure for the chart I'm making this week.

File: src/interface/src/app/treatments/stand-data-chart/stand-data-chart.component.ts (hunk 5 above)
Comment (pflopez) [reply src/interface/src/app/treatments/stand-data-chart/stand-data-chart.component.ts:52]: yep I think a step two is moving some of this stuff to a constant...somewhere. Some sort of base options config for both charts

**Pull request merged**